content/Solutions/Optimizations/Networking.md

```diff
@@ -0,0 +1,339 @@
+---
+title: "Networking Recommendations"
+description: "Best practices for TrueNAS network design."
+weight: 40
+tags:
+ - network
+ - iscsi
+ - interfaces
+---
+
+Follow these best practices for a stable and performant network.
+These generally apply to either TrueNAS CORE or TrueNAS SCALE, unless specified, but each software might place the related options in slightly different web interface locations.
+
+{{< hint type=tip >}}
+iXsystems welcomes contributions from members of the TrueNAS community!
+
+Use the **Feedback** button on the right side or click <svg class="gdoc-icon gdoc_code"><use xlink:href="#gdoc_code"></svg>**Edit Page** at the top right of this page to suggest your own networking tips and tricks.
+{{< /hint >}}
+
+## Static IP Address
+
+By default, TrueNAS SCALE configures the primary network interface for Dynamic Host Configuration Protocol (DHCP) IP address management.
+Consider assigning a static IP address for increased network stability and communication between devices.
+
+{{< include file="/_includes/AliasOrStaticIP.md" >}}
+
+See [Setting Up Static IPs]({{< relref "settingupstaticips.md" >}}) for more information.
+
+## Interfaces
+
+Use multiple network interfaces if possible for load balancing and redundancy.
+Configure link aggregation (LAGG) to combine multiple physical interfaces into a single logical interface.
+
+{{< include file="/_includes/MultipleInterfacesOnNetwork.md" >}}
+
+See also [Setting Up a Link Aggregation]({{< relref "settinguplagg.md" >}}).
+
+### LACP
+
+If supported by your network switch and other networking equipment in your environment, select LACP to use the most common protocol for LAGG interfaces based on [IEEE specification 802.3ad](https://www.ieee802.org/3/hssg/public/apr07/frazier_01_0407.pdf).
+Link Aggregation Control Protocol (LACP) dynamically aggregates multiple network interfaces into a single logical link, providing increased bandwidth and fault tolerance.
+
+LACP allows you to bundle multiple physical network links into a single logical link, increasing the overall bandwidth available.
+This is especially beneficial in environments where high data transfer rates are required, such as when multiple users are accessing storage concurrently.
+LACP does not provide a performance benefit for single-user TrueNAS systems.
```

Comment:
Suggest mentioning that LACP/LAGG is not recommended for iSCSI traffic.

Reply:
https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.storage.doc/GUID-4C19E34E-764C-4069-9D9F-D0F779F2A96C.html

This may be useful here as well.